pkg/apis/core/validation/validation_test.go

```diff
@@ -20572,6 +20573,140 @@ func TestValidateNonSpecialIP(t *testing.T) {
 	}
 }
 
+func TestValidateHostUsers(t *testing.T) {
+	falseVar := false
+	trueVar := true
+
+	cases := []struct {
+		name    string
+		success bool
+		spec    *core.PodSpec
+	}{
+		{
+			name:    "empty",
+			success: true,
+			spec:    &core.PodSpec{},
+		},
+		{
+			name:    "hostUsers unset",
+			success: true,
+			spec:    &core.PodSpec{},
+		},
```

Comment:
Was there something that was suppose to be different about these? Should they be consolidated into a single case?

Reply:
thanks for spotting it.  I'll fix it